tests/cases/employee_health_portal/case_employee_health_portal_request_appointment.yaml

```diff
@@ -75,50 +67,170 @@ TestEmployeeHealthPortalRequestAnAppointmentForUser:
         - Type: eq
           Var: ERROR_MESSAGE
           Value: 'Please select a date. '
-  # filling form
+  # closing form
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.employee_health_telehealth_request_form.cancel_button$
+  # make an appointment
+    - Type: case
+      Value: EmployeeHealthPortalAppointmentsRequest
+  # Verify if the appointment is showing on the list
+    - Type: wait_for
+      Strategy: xpath
+      Id: $PAGE.employee_health_telehealth_request_form.appointment_list$
+    - Type: get_attribute
+      Strategy: xpath
+      Id: $PAGE.employee_health_telehealth_request_form.appointment_list_description$
+      Greps:
+        - var: SHOWED_DESCRIPTION
+          attr: textContent
+          condition: nempty
+          match: "(.*)"
+    - Type: assert
+      Asserts:
+        - Type: eq
+          Var: SHOWED_DESCRIPTION
+          Value: $AND_CLI_PAIN_DESCRIPTION$
+
+#CXTB-113- Review Your Upcoming Appointments & Calendar for Employee EHP User accounts
+TestEmployeeHealthPortalReviewUpcomingAppointment:
+  Environment: Settings
+  Roles: 
+    - Role: desktopChrome
+      App: "desktop"
+  Actions:
+  #Calling Login case
+    - Type: case
+      Value: EmployeeHealthPortalLogIn
+  #Creating many appointment
+    - Type: loop
+      Times: 3
+      Case: EmployeeHealthPortalAppointmentsRequest
+  # get the  upcoming appointment month
+    - Type: get_next_month
+      ResultVar: NEXT_MONTH
+  #View information in "My schedule" tab on portal home page
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.employee_health_home_page.schedule_tab_button$
+    - Type: wait_for
+      Strategy: xpath
+      Id: $PAGE.employee_health_home_page.container_tab$
+  #validating upcoming scheduled event
+    - Type: wait_for
+      Strategy: xpath
+      Id: $PAGE.employee_health_home_page.upcoming_appointment_title$
+    - Type: get_attribute
+      Strategy: xpath
+      Id: $PAGE.employee_health_home_page.upcoming_appointment_title$
+      Greps:
+        - var: MANYAPPOINTMENTS
```

Comment:
please stick with the format separating words using '_'

Reply:
Done